packages/demo-wallet-client/src/components/WarningBanner.tsx

```diff
@@ -1,13 +1,14 @@
+import StellarSdk from "stellar-sdk";
 import { StatusBar } from "@stellar/design-system";
-import { useRedux } from "hooks/useRedux";
+import { getNetworkConfig } from "demo-wallet-shared/build/helpers/getNetworkConfig";
 
 export const WarningBanner = () => {
-  const { account, settings } = useRedux("account", "settings");
+  const { network } = getNetworkConfig();
 
-  // Show the banner only if signed in
-  if (settings.pubnet && account.data?.id) {
+  // Show banner if not on testnet (changed in .env)
+  if (network && network !== StellarSdk.Networks.TESTNET) {
     return (
-      <StatusBar variant={StatusBar.variant.error}>
+      <StatusBar variant={StatusBar.variant.warning}>
         WARNING: You’ve connected a real account to this demo. You are not on
         the test server. Any actions you take here will affect actual assets.
       </StatusBar>
```

Comment:
this may not matter - but this message will show on custom network implementations, as well. Is that intended? Before we were specifically looking for Pubnet

Reply:
Yes, we want to show the message for any network that is not testnet. The message might not be perfect for a custom network, but it should draw attention to any potential risks.